jobs/scheduler/templates/application.properties.erb

```diff
@@ -38,6 +38,9 @@ client.ssl.trust-store=/var/vcap/data/certs/scalingengine/cacerts
 client.ssl.trust-store-password=123456
 client.ssl.protocol=TLSv1.2
 
+#Quartz
+org.quartz.scheduler.instanceName=app-autoscaler
```

Comment:
Created new story for fixing Quartz instance id https://www.pivotaltracker.com/n/projects/1566795
Could you please Approve this if it looks good.Thanks.